antarest/study/storage/variantstudy/model/command/create_binding_constraint.py

```diff
@@ -216,6 +216,12 @@ class AbstractBindingConstraintCommand(OptionalProperties, BindingConstraintMatr
 
     coeffs: t.Optional[t.Dict[str, t.List[float]]] = None
 
+    @model_validator(mode="after")
+    def lower_group(self) -> "AbstractBindingConstraintCommand":
```

Comment:
it looks like we can achieve the same thing with a type annotation:

```python
class Foo(BaseModel):
    bar: Annotated[str, StringConstraints(to_lower=True)]
```
Shouldn't we try to use this where we can instead of having custom validators ?
We can have an alias for this type of course, like `LowerCaseStr`